src/doctopus/db/schema.clj

```diff
@@ -0,0 +1,57 @@
+(ns doctopus.db.schema
+  (:require [clojure.java.jdbc :as sql]
+            [taoensso.timbre :as log]
+            [doctopus.configuration :refer [server-config]]))
+
+(defn- get-subname
+  "pull database options out of the local config"
+  []
+  (let [{host :host port :port db :db} (:database (server-config))]
+    (str "//" (or host "localhost") ":" (or port "5432") "/"
+         (or db "doctopus"))))
+
+(def db-spec {:classname "org.postgresql.Driver"
+              :subprotocol "postgresql"
+              :subname (get-subname)
+              :user (:user (:database (server-config)))
+              :password (:password (:database (server-config)))})
+
+(defn table-created?
+  "check our database for a table"
+  [table-name]
+  (-> (sql/query db-spec [(str "select count(*) from information_schema.tables "
+                           "where table_name='" table-name "'")])
+      first :count pos?))
+
+(def head-schema
+  [[:name :varchar "PRIMARY KEY"]
+   [:created :timestamp "NOT NULL"]
```

Comment:
Got it. W/r/t my comment about letting the DB do some of this: turns out the Clojure JDBC wrapper is just passing db-specific def strings in without modifying them. So, this would be `[:created :timestamp "NOT NULL DEFAULT NOW()"]`, I believe. 

But, I guess I'm also open to not having the DB do this bit? Thoughts?